_deployment/configuration/tds_progman_configuration.md

```diff
@@ -0,0 +1,132 @@
+datasource.acquireIncrement=5
+datasource.acquireRetryAttempts=5
+datasource.checkoutTimeout=60000
+datasource.driverClassName=com.mysql.jdbc.Driver
+datasource.idleConnectionTestPeriod=14400
+datasource.maxConnectionAge=0
+datasource.maxPoolSize=24
+datasource.maxStatements=20000
+datasource.maxStatementsPerConnection=100
+datasource.minPoolSize=12
+datasource.numHelperThreads=3
+datasource.password=<CHANGE ME>
+datasource.testConnectionOnCheckin=false
+datasource.testConnectionOnCheckout=false
+datasource.url=jdbc:mysql://<CHANGE ME>/session
+datasource.username=remoteuser
+dbLockRetryAttemptMax=500
+dbLockRetrySleepInterval=116
+EncryptionKey=Thisisanincrediblylongkeythatiscertainlylongerthantwentyfourcharacters
+iris.ContentPath=/usr/local/tomcat/content/
+itemScoring.callbackUrl=http://requestb.in/16hqwk61
+itemscoring.qti.sympyMaxTries=3
+itemscoring.qti.sympyServiceUrl=http://tds-equation-scoring-service/
+itemscoring.qti.sympyTimeoutMillis=10000
+logLatencyInterval=55
+logLatencyMaxTime=30000
+mna.logger.level=ERROR
+mna.mnaUrl=http://<mna-context-url>/mna-rest/
+mna.oauth.client.id=mna
+mna.oauth.client.secret=<CHANGE ME>
+mnaNodeName=dev
+mnaServerName=proctor_dev
+oauth.testreg.client.granttype=<CHANGE ME>
+oauth.testreg.client.secret=<CHANGE ME>
+oauth.testreg.client=pm
+oauth.testreg.client.id=pm
+oauth.testreg.password=<CHANGE ME>
+oauth.testreg.username=<CHANGE ME>
+oauth.tsb.client.secret=<CHANGE ME>
+oauth.tsb.client=tsb
+opportunity.isScoredByTDS=false
+permission.security.profile=dev
+permission.uri=<CHANGE ME>
+proctor.Appkey=Proctor
+proctor.AppName=Proctor
+proctor.ClientName=SBAC_PT
+proctor.ClientQueryString=false
+proctor.DBJndiName=java:/comp/env/jdbc/session
+proctor.Debug.AllowFTP=true
+proctor.EncryptedPassword=true
+proctor.IsCheckinSite=false
+proctor.ItembankDBName=itembank
+proctor.oauth.checktoken.endpoint=<CHANGE ME>
+proctor.oauth.resource.client.id=pm
+proctor.oauth.resource.client.secret=<CHANGE ME>
+proctor.RecordSystemClient=true
+proctor.security.idp=<CHANGE ME>
+proctor.SessionType=0
+proctor.SqlCommandTimeout=60
+proctor.StateCode=CA
+proctor.TDSArchiveDBName=archive
+proctor.TDSConfigsDBName=configs
+proctor.TDSSessionDBName=session
+proctor.TestRegistrationApplicationUrl=<CHANGE ME>
+student.Appkey=Student
+student.AppName=Student
+student.ClientCookie=true
+student.ClientName=SBAC_PT
+student.ClientQueryString=true
+student.DBDialect=MYSQL
+student.Debug.AllowFTP=true
+student.EncryptedPassword=true
+student.IsCheckinSite=false
+student.oauth.resource.client.secret=<CHANGE ME>
+student.RecordSystemClient=true
+student.SessionType=0
+student.SqlCommandTimeout=60
+student.StateCode=CA
+student.StudentMaxOpps=2
+student.TestRegistrationApplicationUrl=<CHANGE ME>
+student.testScoring.logDebug=true
+student.testScoring.logError=true
+performance.datasource.maxPoolSize=24
+performance.datasource.minPoolSize=12
+performance.logMaxTestOpportunities.enabled=false
+performance.logLatency.enabled=false
+```
+
+
```

Comment:
Maybe put something here about how to use the ProgMan properties after they've been created?  That is, update the `PROGMAN_LOCATOR` in the k8s yaml files or update the tomcat config (which is stored in `/etc/default/tomcat7` file for legacy and in the `setenv.sh` when running locally).   Not sure how valuable that'll be, or if this is even the right spot to tell a user how to update Proctor to use the new ProgMan properties

Reply:
I don't think this is necessarily helpful in this particular document.  This is really just to provide a sample progman configuration.  The deployment yml files call out the information suggested here.   I prefer deployment information to be included in the deployment checklist not in these documents.